controller/array_action/array_mediator_ds8k.py

```diff
@@ -292,19 +297,19 @@ def _extend_volume(self, api_volume, new_size_in_bytes):
             if ERROR_CODE_EXPAND_VOLUME_NOT_ENOUGH_EXTENTS in str(ex.message).upper():
                 raise array_errors.NotEnoughSpaceInPool(api_volume.pool)
 
-    def copy_to_existing_volume_from_source(self, name, source_name, source_capacity_in_bytes,
-                                            minimum_volume_size_in_bytes, pool=None):
+    @convert_scsi_ids_to_array_id(2)
+    def copy_to_existing_volume_from_source(self, volume_id, source_id, source_capacity_in_bytes,
+                                            minimum_volume_size_in_bytes):
         logger.debug(
             "Copy source {0} data to volume {1}. source capacity {2}. Minimal requested volume capacity {3}".format(
-                name, source_name, source_capacity_in_bytes,
+                volume_id, source_id, source_capacity_in_bytes,
```

Comment:
I think the original format here mixed up the target and the source

Reply:
done